chat-plugins/modlog.js

```diff
@@ -333,14 +372,25 @@ exports.commands = {
 			roomIdList = [roomId];
 		}
 
-		PM.send(roomIdList.join(','), searchString, exactSearch, lines).then(response => {
-			connection.send(prettifyResults(response, roomId, searchString, exactSearch === '1', addModlogLinks, hideIps, lines));
+		PM.send(roomIdList.join(','), searchString, lines).then(response => {
+			connection.send(prettifyResults(response, roomId, searchString, response.charAt(0), addModlogLinks, hideIps, lines));
 			if (cmd === 'timedmodlog') this.sendReply(`The modlog query took ${Date.now() - startTime} ms to complete.`);
 		});
 	},
-	modloghelp: [
-		"/modlog [roomid], [search] - Searches the moderator log - defaults to the current room unless specified otherwise.",
-		"If you set [roomid] as [all], it searches for [search] on all rooms' moderator logs.",
-		"If you set [roomid] as [public], it searches for [search] in all public rooms' moderator logs, excluding battles. Requires: % @ * # & ~",
-	],
+	modloghelp: function (target, room, user) {
+		if (!this.runBroadcast()) return false;
+
+		this.sendReplyBox(
+			`/modlog [roomid], [search] - Searches the moderator log - defaults to the current room unless specified otherwise.<br />` +
+			`- If you set [roomid] as [all], it searches for [search] on all rooms' moderator logs.<br />` +
+			`- If you set [roomid] as [public], it searches for [search] in all public rooms' moderator logs, excluding battles. Requires: % @ * # & ~<br />` +
+			`- Put <code style='border: 1px lightblue solid ; background: rgba(0, 50, 0, 0.02) ; padding-left: 3px ; padding-right: 3px'>"</code> around any search term to the exact string.  By default, it will search for full words only.<br />` +
+			`- Use <code style='border: 1px lightblue solid ; background: rgba(0, 50, 0, 0.02) ; padding-left: 3px ; padding-right: 3px'>,</code> include additional search terms.  Results will include lines with all the terms.<br />` +
+			`- Use <code style='border: 1px lightblue solid ; background: rgba(0, 50, 0, 0.02) ; padding-left: 3px ; padding-right: 3px'>|</code> to include additional search terms. Results will include lines with any or all of the terms.<br />` +
+			`- Use <code style='border: 1px lightblue solid ; background: rgba(0, 50, 0, 0.02) ; padding-left: 3px ; padding-right: 3px'>!</code> to exclude any search term.  Results will not include lines containing that term. <br />` +
+			`- Use <code style='border: 1px lightblue solid ; background: rgba(0, 50, 0, 0.02) ; padding-left: 3px ; padding-right: 3px'>*</code> as a wildcard within full word search terms.<br />` +
+			`- Use <code style='border: 1px lightblue solid ; background: rgba(0, 50, 0, 0.02) ; padding-left: 3px ; padding-right: 3px'>lines=#</code> anywhere within the query to choose how many lines of the modlog to display.<br />` +
```

Comment:
Is all this inline CSS really necessary for this? This also isn't usually how we space inline CSS, either

Reply:
I assume no space before the semicolon?  The inline CSS is there to format the ``<code>`` tags to appear somewhat similar to the in-chat code tags on PS.  If there's a CSS class for that, please let me know - I searched the client code, but I didn't see any CSS classes specifically for ``<code>`` tags.

<sub>(Habit of seeing inline CSS spaced this way on Azure - /me hides)</sub>

Reply:
Could just do ``<div class="chat"><code>[text here]</code></div> I guess?

There should probably be an easier way to do this. Maybe you should add an alias to this CSS property on the client :D